src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -95,13 +95,21 @@ public void removePerson(Event key) {
     }
 
     /**
-     * Removes {@code key} from this {@code AddressBook}.
+     * Marks {@code key} from this {@code AddressBook}.
      * {@code key} must exist in the address book.
      */
     public void markEvent(Event target, Event markedEvent) {
         persons.mark(target, markedEvent);
     }
 
+    /**
+     * Unmarks {@code key} from this {@code AddressBook}.
+     * {@code key} must exist in the address book.
+     */
+    public void unmarkEvent(Event target, Event unmarkedEvent) {
```

Comment:
Similarly, can we rename the second param to `eventToUnmark` for easier understanding?